src/Menu/MenuItem.js

```
@@ -1,3 +1,5 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
```

css prop silently fails without these